src/main/java/seedu/address/logic/commands/BrightnessCommand.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.core.Config.ASSETS_FILEPATH;
+import static seedu.address.commons.core.Config.TEMP_FILEPATH;
+
+import java.io.File;
+import java.util.OptionalDouble;
+
+import com.sksamuel.scrimage.BufferedOpFilter;
+import com.sksamuel.scrimage.Image;
+import com.sksamuel.scrimage.filter.BrightnessFilter;
+import com.sksamuel.scrimage.nio.JpegWriter;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.CommandHistory;
+import seedu.address.model.Model;
+
+/**
+ * This command allows users to adjust the brightness of images.
+ */
+public class BrightnessCommand extends Command {
+
+    public static final String COMMAND_WORD = "brightness";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+        + ": Adjust the brightness of the image according to the operators.\n"
+        + "If ratio is not given, default brightness ratio will be 1.1\n"
+        + "[BRIGHTNESS RATIO (double)] "
+        + "and FILENAME.\n"
+        + "Example: " + COMMAND_WORD + " cutedog.jpg"
+        + "Example2: " + COMMAND_WORD + " 0.3 cutedog.jpg";
+    private OptionalDouble brightnessValue;
+    private String fileName;
+
+    /**
+     * Creates a ContrastCommand object.
+     * @param brightnessValue brightness value to add on image
+     * @param fileName is the file name of the image.
+     */
+    public BrightnessCommand(OptionalDouble brightnessValue, String fileName) {
+        this.brightnessValue = brightnessValue;
+        this.fileName = fileName;
+    }
+    @Override
+    public CommandResult execute(Model model, CommandHistory history) {
+        if (this.brightnessValue.isPresent()) {
+            seedu.address.model.image.Image initialImage = new seedu.address
```

Comment:
You can do an `import seedu.address.model.image.Image` then this will be just `Image intialImage = new Image(...)`